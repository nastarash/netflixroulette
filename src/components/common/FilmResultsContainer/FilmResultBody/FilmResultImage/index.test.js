import React from 'react';
import { FilmResultImage } from './index';

describe('<FilmResultImage />', () => {
  test('should be rendered correcly', () => {
    const filmResultImage = shallow(
      <FilmResultImage />
    );
    expect(filmResultImage).toMatchSnapshot();
  });
});