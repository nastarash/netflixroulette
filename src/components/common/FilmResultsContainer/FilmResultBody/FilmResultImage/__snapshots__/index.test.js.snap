// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmResultImage /> should be rendered correcly 1`] = `
<div>
  <img
    className="film-result-image"
  />
</div>
`;
